security-command-center/snippets/src/test/java/vtwo/MuteFindingIT.java

```diff
@@ -132,9 +142,7 @@ public static void cleanUp() throws IOException {
   public static ListFindingsPagedResponse getAllFindings(String sourceName) throws IOException {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
 
-      ListFindingsRequest request = ListFindingsRequest.newBuilder()
-          .setParent(sourceName)
-          .build();
+      ListFindingsRequest request = ListFindingsRequest.newBuilder().setParent(sourceName).build();
```

Comment:
nit: this line is a bit long; consider putting each method invocation after `newBuilder()` on a separate line.

Reply:
Looks like you fixed this—thanks!